tst/regression/test_suites/output_hdf5/output_hdf5.py

```diff
@@ -81,7 +81,10 @@ def Analyse(self,parameters):
         analyze_status = True
         print(os.getcwd())
 
-        sys.path.insert(1, '../../../../../scripts/python')
+        parthenonPath = os.path.realpath(__file__)
+        idx = parthenonPath.find('/parthenon/')
```

Comment:
Can you use rfind instead:

`
parthenonPath.rfind('/parthenon/')
`

I can see cases where there might be a folder inside of a folder.

Reply:
find will return the first instance.

Reply:
That's a good suggestion: that will also eliminate the '/parthenon/' from the rest of the strings.